cmd/influx_inspect/export.go

```diff
@@ -18,29 +19,54 @@ type cmdExport struct {
 	out             string
 	db              string
 	retentionPolicy string
+	startTime       int64
+	endTime         int64
 	compress        bool
 
 	ext   string
 	files map[string][]string
 }
 
-func newCmdExport(path, out, db, retentionPolicy string, compress bool) *cmdExport {
+func newCmdExport(path, out, db, retentionPolicy, startTime, endTime string, compress bool) (*cmdExport, error) {
+	var start, end int64
+	if startTime != "" {
+		s, err := time.Parse(time.RFC3339, startTime)
+		if err != nil {
+			return nil, err
+		}
+		start = s.UnixNano()
+	}
+	if endTime != "" {
+		e, err := time.Parse(time.RFC3339, endTime)
+		if err != nil {
+			return nil, err
+		}
+		end = e.UnixNano()
+	} else {
+		// set end time to max if it is not set.
+		end = 1<<63 - 1
+	}
```

Comment:
Need a check somewhere to see if end time is less than start time (unless I missed it)